src/backend/marsha/settings.py

```diff
@@ -428,9 +428,9 @@ class Base(Configuration):
 
     # Cors
     CORS_ALLOW_ALL_ORIGINS = values.BooleanValue(False)
-    CORS_ALLOWED_ORIGINS = values.ListValue([])
+    CORS_ALLOWED_ORIGINS = values.ListValue(["http://localhost:3000"])
```

Comment:
You can't modify it here. You can do it in the `Development` class maybe but not here.

Reply:
Sorry, it is not linked with your PR, you can ignore this comment.